missioncontrol/commands/addjpd.go

```diff
@@ -0,0 +1,52 @@
+package commands
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli-go/missioncontrol/utils"
+	"github.com/jfrog/jfrog-cli-go/utils/config"
+	"github.com/jfrog/jfrog-client-go/httpclient"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"net/http"
+)
+
+func AddJpd(flags *AddJpdFlags) error {
+	missionControlUrl := flags.MissionControlDetails.Url + "api/v1/jpds"
+	httpClientDetails := utils.GetMissionControlHttpClientDetails(flags.MissionControlDetails)
+	client, err := httpclient.ClientBuilder().Build()
+	if err != nil {
+		return err
+	}
+	resp, body, err := client.SendPost(missionControlUrl, flags.JpdSpec, httpClientDetails)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode != http.StatusCreated {
+		return errorutils.CheckError(errors.New(resp.Status + ". " + utils.ReadMissionControlHttpMessage(body)))
+	}
+
+	log.Debug("Mission Control response: " + resp.Status)
+	log.Output(clientutils.IndentJson(body))
+	return nil
+}
+
+type AddJpdFlags struct {
+	MissionControlDetails *config.MissionControlDetails
+	JpdSpec               []byte
+}
+
+type AddJpdRequestContent struct {
+	Name     string       `json:"name,omitempty"`
+	Url      string       `json:"url,omitempty"`
```

Comment:
Do these attributes need to be documented? If it isn't already the case, maybe a link to the wiki could be added to the help.

Reply:
The structs are actually no longer needed since the command expects a JSON with all configurations